### ISSUE TITLE
admin_user ??

### ISSUE THREAD

**Issue by rcgoodfellow** [deter-project/nu#3]
When I invoke the command `rake db:setup` i see the following output

```
Logging in as 
D, [2015-08-29T21:00:16.768818 #4960] DEBUG -- : HTTPI GET request to spidev.isi.deterlab.net (wsdl_caching_adapter)
D, [2015-08-29T21:00:16.794529 #4960] DEBUG -- : HTTPI POST request to spidev.isi.deterlab.net (wsdl_caching_adapter)
rake aborted!
Invalid admin credentials
```

How do I set the admin_user ?

**Comment by alg** [comment]
I disabled the seeding upon database setup now, as it doesn't make sense to re-seed the db with everyone's db setup. You should be good even without re-trying the `db:setup` command now as it was the last task in the batch.

**Comment by rcgoodfellow** [comment]
I can verify that this issue is fixed in commit 50fb5dafdef833df940b584737eaa7a5aa12e34a. Thanks @alg !
